docs/_extra/deploying-yaml.md

```diff
@@ -27,6 +27,8 @@ you can:
 $ curl -T ./blueprint.yaml -X POST http://localhost:8081/v1/applications
 {% endhighlight %}
 
+You may also need a `-H "Content-Type: application/yaml"` depending on type configuration.
+(Not usually for this, but often for other calls.)
```

Comment:
i had to dig that out when testing this.  it's in the unused `_extra` folder of docs which we want to fold in elsewhere.